docs/consuming.md

```
@@ -71,6 +71,77 @@ jobs:
+}
+```
+
+`bypass_mode = "always"` is required. The other mode, `pull_request`, only grants the
+bypass to a pull request the actor opened — and the whole difficulty here is that the
+bump has no pull request.
+
+`actor_id` is the **App id**, which the installation endpoint reports (the installation's
+own `.id` is a different number and will not match anything):
+
+```console
+$ gh api /repos/OWNER/REPO/installation --jq .app_id
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)